css/styles.css

```diff
@@ -396,7 +398,7 @@ header .menu-bars:hover {
   background-size: contain;
   background-repeat: no-repeat;
   background-position: right 0;
-  background-color: #0017EB;
+  background-color: #0017eb;
   border-top-left-radius: 3em;
 }
```

Comment:
Kindly make sure to use the right background color as on the Figma template. Your background color should be `#6070FF` instead of `#0017eb`. Please be sure to refer back to the Figma template on every style you add.